app/src/actions/user.js

```diff
@@ -51,6 +51,7 @@ export function logout() {
     dispatch({
       type: LOGOUT
     });
+    window.location.hash = window.location.hash.replace(/#access_token=([a-zA-Z0-9.\-_]*)/g, '');
```

Comment:
I don't get what the `(&[a-z=])?` bit is doing here https://github.com/Vizzuality/GlobalFishingWatch/blob/master/app/src/components/App.jsx#L4
(capturing one single char after & ?)